astropy/modeling/core.py

```diff
@@ -3616,6 +3617,62 @@ def fix_inputs(modelinstance, values):
     return CompoundModel('fix_inputs', modelinstance, values)
 
 
+class _TieParameterTo(Callable):
+    """Tie `~astropy.modeling.Model` `~astropy.modeling.Parameter` to.
```

Comment:
I'm not sure I understand this sentence.

Reply:
thanks. Amending it "Tie `astropy.modeling.Parameter` to another parameter". The `~` is omitted to show the path, as Parameter is a common name.

Reply:
👍 . pushed.